File: src/main/resources/scripts/cppSide/file.py
# arg1 = int(input("arg1"))
# arg2 = int(input("arg2"))
# res = arg1 + arg2
# with open('output.txt', 'w') as file:
#     file.write(str(res))

import sys
import Solution

arg1 = int(sys.argv[1])
arg2 = int(sys.argv[2])
res = Solution.twoSum(arg1, arg2)
with open('output.txt', 'w') as file:
    file.write(str(res))

File: src/main/resources/scripts/cppSide/file.js
const { foo } = require('./Solution');

function output() {
    const num1 = process.argv[2];
    const num2 = process.argv[3];

    console.log(`You entered: ${num1} and ${num2}`);
    const sum = foo(parseInt(num1), parseInt(num2));
    console.log(`The sum is: ${sum}`);
    const fs = require('fs');
    const data = sum.toString();
    fs.writeFile('output.txt', data, (err) => {
        if (err) throw err;
        console.log('Data written to file');
    });
}

output();